charts/vald/values.yaml

```
@@ -539,14 +542,21 @@ defaults:
             dual_stack_enabled: true
           # @schema {"name": "defaults.grpc.client.dial_option.tcp.tls", "alias": "tls"}
           tls:
-            # defaults.grpc.client.dial_option.tcp.tls.enabled -- gRPC client TCP TLS enabled
+            # @schema {"name": "defaults.grpc.client.dial_option.tcp.tls.enabled", "type": "boolean"}
+            # defaults.grpc.client.dial_option.tcp.tls.enabled -- TLS enabled
             enabled: false
-            # defaults.grpc.client.dial_option.tcp.tls.cert -- gRPC client TCP TLS cert path
+            # @schema {"name": "defaults.grpc.client.dial_option.tcp.tls.cert", "type": "string"}
+            # defaults.grpc.client.dial_option.tcp.tls.cert -- TLS cert path
             cert: /path/to/cert
-            # defaults.grpc.client.dial_option.tcp.tls.key -- gRPC client TCP TLS key path
+            # @schema {"name": "defaults.grpc.client.dial_option.tcp.tls.key", "type": "string"}
+            # defaults.grpc.client.dial_option.tcp.tls.key -- TLS key path
             key: /path/to/key
-            # defaults.grpc.client.dial_option.tcp.tls.ca -- gRPC client TCP TLS ca path
+            # @schema {"name": "defaults.grpc.client.dial_option.tcp.tls.ca", "type": "string"}
+            # defaults.grpc.client.dial_option.tcp.tls.ca -- TLS ca path
             ca: /path/to/ca
+            # @schema {"name": "defaults.grpc.client.dial_option.tcp.tls.insecure_skip_verify", "type": "boolean"}
+            # defaults.grpc.client.dial_option.tcp.tls.insecure_skip_verify -- enable/disable skip SSL certificate verification
+            insecure_skip_verify: false
```

Since `"alias": "tls"` (refers L397) is used in this section, `@schema` annotations in these lines are useless.
Also for the other sections `tls` below in this file.

```suggestion
            # defaults.grpc.client.dial_option.tcp.tls.enabled -- TLS enabled
            enabled: false
            # defaults.grpc.client.dial_option.tcp.tls.cert -- TLS cert path
            cert: /path/to/cert
            # defaults.grpc.client.dial_option.tcp.tls.key -- TLS key path
            key: /path/to/key
            # defaults.grpc.client.dial_option.tcp.tls.ca -- TLS ca path
            ca: /path/to/ca
            # defaults.grpc.client.dial_option.tcp.tls.insecure_skip_verify -- enable/disable skip SSL certificate verification
            insecure_skip_verify: false
```